systems/edge-nodes/modules/caddy.nix

```
@@ -139,6 +139,23 @@
         # <https://caddyserver.com/docs/caddyfile/options#trusted-proxies-strict>
         trusted_proxies_strict
       }
+      (cors) {
+        @cors_preflight method OPTIONS
+        @cors header Origin {args.0}
+
+        handle @cors_preflight {
+          header Access-Control-Allow-Origin "{args.0}"
+          header Access-Control-Allow-Methods "GET, POST, PUT, PATCH, DELETE"
+          header Access-Control-Allow-Headers "Content-Type"
+          header Access-Control-Max-Age "3600"
+          respond "" 204
+        }
+
+        handle @cors {
+          header Access-Control-Allow-Origin "{args.0}"
```

```suggestion
        @cors header Origin {args[0]}

        handle @cors_preflight {
          header Access-Control-Allow-Origin "{args[0]}"
          header Access-Control-Allow-Methods "GET, POST, PUT, PATCH, DELETE"
          header Access-Control-Allow-Headers "Content-Type"
          header Access-Control-Max-Age "3600"
          respond "" 204
        }

        handle @cors {
          header Access-Control-Allow-Origin "{args[0]}"
```
deprecated syntax, everything else is fine